docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,120 @@
+Migrating Your Existing Workflows to Use GitHub Apps
+======================================================
+
+Dockstore 1.9.0 provides users with a way to keep their workflows automatically updated (instead of needing to manually refresh) by using GitHub apps.
+Here, we will go over how to migrate your existing Dockstore workflows to use GitHub apps. Also keep in mind that the migration process is done one branch/version at a time.
+This tutorial assumes that you are are familiar with the  ``/.dockstore.yml`` file by having read our overview of :doc:`GitHub apps </getting-started/github-apps>`. and the :ref:`Registration With GitHub Apps` section
+in our workflow registration document.
+
+GitHub App Installation
+-----------------------
+
+The first step to migrating a workflow is the same as adding a new workflow via GitHub apps: install our Dockstore GitHub app onto your repository or
+organization. You do this by navigating to ``/my-workflows``, clicking the ``+`` button on the left hand sidebar, selecting ``Register using GitHub Apps``, and then clicking
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
+
+.. image:: /assets/images/docs/add-workflow-button.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/register-workflow-github-apps.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/gh-app-reg-1.png
+   :width: 40 %
+
+Creating a .dockstore.yml File
+-------------------------------
+
+Once the GitHub app is installed on the correct repo, the next step is to create a ``/.dockstore.yml`` file. We'll cover a very straigtforward example
+first, but depending on how you configured the workflow during registration and whether your GitHub repository houses multiple workflows published on Dockstore,
+there will be additional steps to writing your ``/.dockstore.yml`` file.
+
+Let's say we have the following CWL workflow registered on Dockstore that came from this `repository <https://github.com/NatalieEO/ghapps-single-workflow>`__ and you would like to convert the master branch.
+
+.. figure:: /assets/images/docs/single-workflow-to-migrate.png
+   :alt: Workflow to Migrate
+
+As noted in our other documentation, create a ``/.dockstore.yml`` file in the root directory of the master branch in your repository. The file should look like the following
```

Comment:
I'm worried users might think this means they have to use a master branch to do the migration, though I'm not sure how else to word this.